lib/Lib.RandomBuffer.System.fsti

```diff
@@ -2,13 +2,40 @@ module Lib.RandomBuffer.System
 
 open Lib.IntTypes
 open Lib.Buffer
+module B = LowStar.Buffer
 
 open FStar.HyperStack.All
 
+open Lib.RandomSequence
+module R = Lib.RandomSequence
 
+/// WARNING: This function is DEPRECATED. Use ``random_crypto`` instead.
```

Comment:
I think there is a kremlin macro to deprecate functions, isn't it @protz ?

Reply:
yes! `[@@ deprecated "random_crypto" ]` which will give a deprecation warning in F* and in C as well via compiler-specific pragmas